packages/webapp/src/components/Filter/FilterButton/styles.module.scss

```diff
@@ -0,0 +1,50 @@
+.filterButtonContainer {
+  position: relative;
+  height: 100%;
```

Comment:
In Figma and in Storybook this results in a square button, but in app the button ends up rectangular in both Finances and the old PureSearchbarAndFilter (I think as height will be more than 40px). I'm okay with either and I'll match the search button, but just checking that it shouldn't stay square.

Reply:
I kinda prefer it to be square too, I'll update!